common/persistence/persistence-tests/executionManagerTest.go

```diff
@@ -1171,7 +1171,7 @@ func (s *ExecutionManagerSuite) TestPersistenceStartWorkflow() {
 
 	task1, err1 := s.CreateWorkflowExecution(ctx, domainID, workflowExecution, "queue1", "wType1", 20, 14, nil, 3, 0, 2, nil, nil)
 	s.Error(err1, "Expected workflow creation to fail.")
-	log.Infof("Unable to start workflow execution: %v", err1)
+	log.Printf("Unable to start workflow execution: %v\n", err1)
```

Comment:
printf's fine / no need to change, but this is equivalent to the somewhat-simpler:
```
log.Println("Unable to start workflow execution:", err1)
```

Reply:
we can just use `s.T().Logf` in tests